### PR TITLE
sophiapp: add version 1.0.82

### LINKED ISSUE #9209: [Request] add SophiApp
### Criteria

For a package to be acceptable in the Extras bucket, it should be:

- [x] a well-known and widely used tool (e.g. if it's a GitHub project, it should have at least 100 stars and 50 forks)
- [x] the latest stable version of the program

### Information

**Name:** [name of new package]
sophiapp

**Description:** [clear and concise details of what it is]
The most powerful open source tweaker for Windows 10 & 11.

**Homepage:** [a URL/link]
https://github.com/Sophia-Community/SophiApp

**Download link(s):** [URL(s)/link(s)]
https://github.com/Sophia-Community/SophiApp/releases/download/1.0.82/SophiApp.zip

**Some indication of popularity/repute:** [GitHub stars/software reviews etc.]
https://github.com/Sophia-Community/SophiApp

### PR DESCRIPTION
**Pull request by KrisSchneider**
Closes #9209

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

Hello, this is my first time adding a manifest to the official ScoopInstaller/Extras repository. What mistakes did I make?

### CODE REVIEW

**Comment by KrisSchneider** [comment]
what did I do wrong?

**Comment by KrisSchneider** [comment]
Whoever is online right now, can you please help me? I am having problem with auto updating for my manifest.

**Comment by KrisSchneider** [comment]
When I update the app using this command, `scoop update sophiapp -f`. This is what happens:
**Before**
image_3

**After**
image_4

Could someone please explain to me what I'm doing wrong for the manifest file?

**Comment by KrisSchneider** [comment]
Never mind, I learned how to fix my problem. I'm good now.

/verify

File: bucket/sophiapp.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.0.82",
+    "description": "The most powerful open source tweaker for Windows 10 & 11.",
```
Comment (issaclin32) [inline bucket/sophiapp.json:3]:
```suggestion
    "description": "An open-source app for configuring and fine-tuning Windows 10 & Windows 11.",
```
I think `The most powerful open source tweaker` looks vague and doesn't really tell us what the app is doing (although its on their Github repo descrition.)

File: bucket/sophiapp.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.0.82",
+    "description": "The most powerful open source tweaker for Windows 10 & 11.",
+    "homepage": "https://github.com/Sophia-Community/SophiApp/",
+    "license": "MIT",
+    "notes": [
+        "Be careful when using this app, you can cause some serious harm to your computer."
+    ],
```
Comment (issaclin32) [inline bucket/sophiapp.json:8]: My opinion is that if this app has potential problem to **break the system**, then we shouldn't add this to the official buckets.

Comment (KrisSchneider) [reply bucket/sophiapp.json:8]: I tested it again on my system, I was wrong about it harming my computer. It turns out, it just closes some processes such as, _explorer.exe_, and _everything.exe_ while it applies the tweaks that you want. 

I will remove will note now, consider this a false warning from my end. No need to worry.

**Comment by KrisSchneider** [comment]
@issaclin32 Do you know why my **Autoupdate Hash Extraction** is not working?

File: bucket/sophiapp.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "extract_dir": "SophiApp",
+    "shortcuts": [
+        [
+            "SophiApp.exe",
+            "SophiApp"
+        ]
+    ],
+    "post_install": [
+        "if (Test-Path \"$dir\\SophiApp.exe.config.original\") {Remove-Item -Path \"$dir\\SophiApp.exe.config.original\" | Out-Null}"
+    ],
```
Comment (issaclin32) [inline bucket/sophiapp.json:17]:
```suggestion
```

We don't need to remove this, unless this is causing the app fail to start.

File: bucket/sophiapp.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "persist": [
+        "SophiApp.exe.config",
+        "Logs"
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/Sophia-Community/SophiApp/releases/download/$version/SophiApp.zip",
+        "hash": {
```
Comment (issaclin32) [inline bucket/sophiapp.json:25]:
```suggestion
        "hash": {
```
The trailing whitespace is causing CI to fail

**Comment by issaclin32** [comment]
https://github.com/ScoopInstaller/Extras/pull/9210#discussion_r968144458

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**